### PR TITLE
[16.0][MIG] stock_inventory_line_product_cost

### PR DESCRIPTION
**Pull request by Urvisha-OSI**

### CODE REVIEW

File: stock_inventory_line_product_cost/README.rst [hunk 1]
```diff
@@ -19,3 +19,4 @@ Contributors
 ------------
 
 * Open Source Integrators <contact@opensourceintegrators.com>
+* Urvisha Desai <udesai@opensourceintegrators.com>
```
Comment (zamberjo) [inline stock_inventory_line_product_cost/README.rst:22]: Do not modify the Readme directly, as it is self-generated, but create the README folder.
SEE: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Review by zamberjo — COMMENTED** [review]
Technical review seems ok, can you look at the README and do a rebase to get the runboat build done so someone can review it functionally?

File: stock_inventory_line_product_cost/README.rst (hunk 1 above)
Comment (rousseldenis) [reply stock_inventory_line_product_cost/README.rst:22]: @zamberjo That's only the case when readme fragments are available. This is not the case

**Pull request closed without merging**